ci/gpu/build.sh

```diff
@@ -269,18 +269,27 @@ cd "$WORKSPACE/python/custreamz"
 gpuci_logger "Python py.test for cuStreamz"
 py.test -n 8 --cache-clear --basetemp="$WORKSPACE/custreamz-cuda-tmp" --junitxml="$WORKSPACE/junit-custreamz.xml" -v --cov-config=.coveragerc --cov=custreamz --cov-report=xml:"$WORKSPACE/python/custreamz/custreamz-coverage.xml" --cov-report term custreamz
 
+
+# only install strings_udf after cuDF is finished testing without its presence
 gpuci_logger "Installing strings_udf"
 gpuci_mamba_retry install strings_udf -c "${CONDA_BLD_DIR}" -c "${CONDA_ARTIFACT_PATH}"
 
 # only install strings_udf after cuDF is finished testing without its presence
 cd "$WORKSPACE/python/strings_udf/strings_udf"
 gpuci_logger "Python py.test for strings_udf"
 
-# retest cudf with strings_udf present
-cd $WORKSPACE/python/cudf/cudf
-py.test -n 8 --cache-clear --basetemp="$WORKSPACE/strings-udf-cuda-tmp" --junitxml="$WORKSPACE/junit-strings-udf.xml" -v --cov-config=.coveragerc --cov=strings_udf --cov-report=xml:"$WORKSPACE/python/strings_udf/strings-udf-coverage.xml" --cov-report term tests
-gpuci_logger "Python py.test retest cuDF UDFs"
-py.test tests/test_udf_masked_ops.py -n 8 --cache-clear
+STRINGS_UDF_PYTEST_RETCODE=0
```

Comment:
I'm confused, I remember writing this logic (to avoid failing when no tests are run) in the original strings UDF PR. Did it get removed?

Reply:
Ah I see, once we enabled CEC we no longer need to handle that case, and now we have to again. Carry on.